components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.m

```diff
@@ -250,51 +219,35 @@ - (void)setTitleColor:(UIColor *)titleColor {
   _titleLabel.textColor = titleColor;
 }
 
+- (UIFont *)defaultTitleFont {
```

Comment:
Make into a class method.

Reply:
Should this be a readonly class property instead of a method?  Do we have a preference either way?

Reply:
Property FTW.